ublox/src/ubx_packets/packets.rs

```diff
@@ -8,6 +8,7 @@ use chrono::prelude::*;
 use core::fmt;
 use num_traits::cast::{FromPrimitive, ToPrimitive};
 use num_traits::float::FloatCore;
+use num_traits::real::Real;
```

Comment:
Is this needed?  Maybe by the generated code? If so the generated code should probably be emitting code that uses the trait like 

```rust
fn example(f: impl ::num_traits::real::Real) {
    let a = ::num_traits::real::Real::abs(f);
}
``` 

this makes the macro most "hygienic" - https://gist.github.com/Kestrer/8c05ebd4e0e9347eb05f265dfb7252e1#use-unambiguous-traits

this is not super important because this macro is only used internally to this crate. so not worth fixing right now